src/Pillar-Chrysal/ConfigurationForPillar.class.st

```diff
@@ -384,6 +384,11 @@ ConfigurationForPillar >> declaredItems [
 	^ #( #bibFile #title #attribution #headingLevelOffset #series #keywords #outputDirectory #baseDirectory #outputFile #mainDocument #inputFile #latexTemplate #scrambledEmailAddresses #separateOutputFiles #renderStructureAsSlide #latexChapterTemplate #htmlTemplate #htmlChapterTemplate #newLine #latexWriter #htmlWriter #slideInTemplateForDeckJS #defaultScriptLanguage #levelSpecification #levels #tocFile )
 ]
 
+{ #category : #'as yet unclassified' }
+ConfigurationForPillar >> defaultExport [
+	^ self propertyAt: #defaultExport ifAbsent: [ '' ].
```

Comment:
I don't think returning an empty string is a good solution.
- If there is no property `#defaultExport` we should have an error.
- in addition you could write a method that receives a block argument!

Something like

```smalltalk
defaultExportIfNone: aBlock [
  ^ self propertyAt: #defaultExport ifAbsent: aBlock
]
```